corehq/apps/reports/tasks.py

```diff
@@ -370,9 +370,9 @@ def _expose_download(fpath, use_transfer, zip_name, download_id, num_forms):
 
 
 def _format_filename(form_info, question_id, extension, case_id_to_name):
-    filename = u"{}-user_{}-form_{}{}".format(
+    filename = u"{}-{}-form_{}{}".format(
         unidecode(question_id),
-        form_info['form'].user_id or 'unknown',
+        form_info['form'].get_data('form/meta/username') or form_info['form'].user_id or 'user_unknown',
```

Comment:
right, makes sense. In that case it may make sense to try and cache the username by user_id's so that you don't have to fetch the form XML for every form (unless you're already having to fetch it for some other reason)